e2e/etcd_test.go

```diff
@@ -402,3 +408,7 @@ func (epc *etcdProcessCluster) endpoints() []string {
 	}
 	return eps
 }
+
+func (epc *etcdProcessCluster) dataDir(i int) string {
```

Comment:
I don't think this is necessary since `backends()[0].cfg.dataDirPath` does the same thing